spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -61,6 +61,39 @@ import org.apache.comet.shims.CometExprShim
  * An utility object for query plan and expression serialization.
  */
 object QueryPlanSerde extends Logging with CometExprShim {
+
+  /**
+   * Mapping of Spark expression class to Comet expression handler.
+   */
+  private val exprHandlers: Map[Class[_], CometExpressionSerde] = Map(
```

Comment:
Thats neat

Reply:
Prob we can call `exprMappers` or `sparkToCometExprMapper` ?

Reply:
I renamed to `exprSerdeMap` since the map holds `CometExpressionSerde` instances. I am open to other suggestions though,